ansible/kolla-openstack.yml

```diff
@@ -94,6 +94,7 @@
     switch_type_to_device_type:
       arista: netmiko_arista_eos
       dellos9: netmiko_dell_force10
+      dellos10: netmiko_dell_os10
```

Comment:
I assume we already have this patch but this was omitted from the backport?

Reply:
I think the issue was that we needed dell os 10 support to land in NGS upstream, but that only landed in 2023.1. So we were missing that part of the dellos10 support in kayobe i.e we only supported physical network configure.